Jasmine/Common/Models/Levels.swift

```diff
@@ -40,10 +57,44 @@ class Levels {
         }
     }
 
+    /// Returns a valid name with whitespaces trimmed
+    ///
+    /// - Parameter name: original name
+    /// - Returns: name with whitespace trimmed, default if no name passed or trimmed name is empty
+    /// - Throws: LevelsError.duplicateLevelName if name already exists
+    private func getValidName(_ name: String?) throws -> String {
+        var trimmedName = name?.trimmingCharacters(in: .whitespaces)
+        if let isNameEmpty = trimmedName?.isEmpty, isNameEmpty {
```

Comment:
Shorten it to something nicer. 
```let name = (trimmedName?.isEmpty ?? true) ? nextAvailableDefaultName : trimmedName``` would be perfect but it returns an optional even though that optional is definitely non-nil...